### PR TITLE
Docu updated

### PR DESCRIPTION
**Pull request by staehler**
I tried my best to describe syncing the repos (official - private - local), but it might be not correct. I've written down my personal and unfortuantely first attempt to sync the repos. So please have a look at it and correct where ever you mean. As I found some github screen snapshots on CometVisu\doc\manual\de\install\_static you should include them in the docu at the right position, they could help. 
I've left some word about eibd and knxd installation, but there might be better docu already available, but hasn't moved yet. Feel free to dismiss my additions, if not worth.
Some screen snapshots added as pictures say more than words.

Regards,
Michael

### CODE REVIEW

File: doc/manual/de/config/widgets/include/index.rst [hunk 1]
```diff
@@ -31,4 +31,6 @@ Wie im Beispiel zu sehen, kann man z.B. für jeden Raum eine eigene XML-Datei an
     die index.html liegt). Wenn die einzubindende Datei im config-Ordner liegt, muss also ``src=config/<dateiname>``
     angegeben werden (siehe Beispielcode oben).
 
+.. hint::
+    Der Abschnitt zwischen <meta> ... </meta> kann nicht ausgelagert werden.
```
Comment (peuter) [inline doc/manual/de/config/widgets/include/index.rst]: Der XML-Code sollte in zwei Backticks eingefasst werden, um ihn auch als Code zu kennzeichnen

File: doc/manual/de/install/backends/install-eibd.rst [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+Installation von eibd
+---------------------
+Der eibd Service verbindet den Computer mit dem KNX/EIB Bus und unterstützt Schnittstellen wie USB, TPUART oder EIBnet/IP Routing and Tunneling. eibd ist eine Entwicklung der TU Wien  und wird nicht weiter entwickelt. Dennoch ist der eibd noch häufig im Einsatz und ein stabiler Service. Der Link zur offiziellen Installationsanleitung 
+-  `deb Pakete <https://www.auto.tuwien.ac.at/~mkoegler/index.php/eibdeb>`__
+-  `rpm Pakete <https://www.auto.tuwien.ac.at/~mkoegler/index.php/eibrpm>`__ 
+-  `nslu2 <https://www.auto.tuwien.ac.at/~mkoegler/index.php/eibnslu>`__
+
+Installation von knxd
+---------------------
+Aufgrund des Entwicklungsstillstands bei eibd wurde der Fork knxd gebildet. knxd ist ein KNX Router, der auf allen Linux Computern läuft und kann mit allen bekannten KNX Schnittstellen kommunizieren.
```
Comment (peuter) [inline doc/manual/de/install/backends/install-eibd.rst]: "und mit allen bekannten KNX Schnittstellen kommunizieren kann" klingt für mich besser

File: doc/manual/de/config/widgets/include/index.rst (hunk 1 above)
Comment (staehler) [inline doc/manual/de/config/widgets/include/index.rst]: Werde ich anpassen.

File: doc/manual/de/install/backends/install-eibd.rst (hunk 2 above)
Comment (staehler) [inline doc/manual/de/install/backends/install-eibd.rst]: Werde ich anpassen.

**Comment by peuter** [comment]
Danke @staehler

**Pull request merged**